apps/shop/views.py

```diff
@@ -125,3 +114,37 @@ def list(self, request):
             raise APIException
         serializer = ShopContactsSerializer(queryset)
         return Response(serializer.data)
+
+
+class ReviewViewSet(viewsets.ViewSet):
```

Comment:
Didn't you wanted to make it ModelViewSet?

Reply:
because if you are going to use ModelViewSet with permission_classes=[IsAdminUser], it  applies to the entire class. also on reviews_by_item()